src/components/list/listItem/subitem/ExpandableTextArea.tsx

```diff
@@ -63,3 +75,6 @@ const ExpandableTextArea: React.FC<ExpandableTextAreaProps> = ({
 };
 
 export default ExpandableTextArea;
+function enrchHTML(arg0: string) {
```

Comment:
I believe this function is auto-generated by IDE when you click "create function" accidently. Could you remove it?

Reply:
removed